src/Mod/BIM/bimcommands/BimTDView.py

```diff
@@ -38,7 +38,7 @@ def GetResources(self):
             "MenuText": QT_TRANSLATE_NOOP("BIM_TDView", "Insert view"),
             "ToolTip": QT_TRANSLATE_NOOP(
                 "BIM_TDView",
-                "Inserts a drawing view on a page",
+                "Inserts a drawing view on a page.\nTo choose where to insert the view when multiple pages are available, \nselect both the view and the page before you execute the command.",
```

Comment:
This line is rather long, how about splitting it up like this:
```suggestion
                "Inserts a drawing view on a page.\n"
                "To choose where to insert the view when multiple pages are available,\n"
                "select both the view and the page before you execute the command.",
```
What do you think?

Reply:
Sounds good. I simply went with the same formatting style as the other commands in the BIM/Draft code, but I do prefer the splitting for readability.